### PR TITLE
Update site logo

### PR DESCRIPTION
**Pull request by mildronize**
Please review my PR.

### CODE REVIEW

File: src/components/Header/logo.js [hunk 1]
```diff
@@ -55,17 +55,20 @@ const LogoWrapper = styled.div`
...
-  split[last] = split[last] + '</strong>';
-  const title2 = split.join(' ');
+  /*
+    Note: Disable first word bold
+  */
+  // let split = title.split(' ');
+  // split[0] = '<strong>' + split[0];
+  // const last = split.length < 3 ? 0 : split.length - 2;
+  // split[last] = split[last] + '</strong>';
+  // const title2 = split.join(' ');
```
Comment (aaronamm) [inline src/components/Header/logo.js]: Do we still need this code? If not, could you please remove this code comment block?

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -229,7 +229,7 @@ const Header = ({ setShowSearch, location, themeProvider, show, toggleFullscreen
         },
       } = data;
       const logoLink = logo.link !== '' ? logo.link : '/';
-      const logoImg = require('images/logo.svg');
+      const logoImg = require('images/dotnetthailand-logo.svg');
```
Comment (aaronamm) [inline src/components/Header/Header.js]: Just curiosity, does import statement work for import resource file?
If so, could you please use import statement?

**Comment by aaronamm** [comment]
@mildronize Thank you so much I appreciate your help.

File: src/components/Header/logo.js (hunk 1 above)
Comment (mildronize) [reply src/components/Header/logo.js]: OK.

**Review by aaronamm — APPROVED** [review]

**Pull request merged**